issue_12/script.js

```diff
@@ -0,0 +1,75 @@
+function startLoading() {
+    const loadingImg = document.createElement('img');
+    loadingImg.src = 'loading-circle.gif';
+    loadingImg.alt = 'ローディング画像';
+    document.getElementById('js-loading').appendChild(loadingImg);
+}
+
+function removeLoading() {
+    document.getElementById('js-loading').remove();
+}
+
+function createLists(resolvedData) {
```

Comment:
IMO
I think `render` is better here too.
```suggestion
function renderLists(resolvedData) {
```

Reply:
Fixed this one too!
https://github.com/yoshikawa-akr/js-lessons/pull/12/commits/79b7de8f090338f4b10594ae1655b88aa7dc0ee3